Added Ports Explanation to Prerequisites section (#403)
subspace/subspace-docs: docs/farming-&-staking/farming/additional-guides/network-info.md

| Port  | Protocol            | Exposure                                                | Short Description           | Used By                                                                                                                                                              | Purpose                                              |
|-------|---------------------|---------------------------------------------------------|----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
| 30333 | TCP/UDP (QUIC)      | Always public                                           | Substrate networking P2P   | <ul><li>[Pulsar](../../../category/pulsar/)</li><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li><li>[Space Acres](https://github.com/nazar-pc/space-acres)</li></ul> | Connecting to the rest of the network for exchanging blocks, transactions, PoT gossip, etc. |

> | 30433 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Pulsar](../../../category/pulsar/)</li><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li><li>[Space Acres](https://github.com/nazar-pc/space-acres)</li></ul> | Connecting to the rest of the network and retrieve archival history            |

Reviewer: ```suggestion
| 30433 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Pulsar](../../../category/pulsar/)</li><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li><li>[Space Acres](https://github.com/nazar-pc/space-acres)</li></ul> | Connecting to the rest of the network and retrieving archival history.            |
```

> | 30533 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li></ul>                                                                        | Connecting to the rest of the network and retrieve archival history, not needed for Pulsar and Space Acres because node and farmer share Subspace networking stack there      |

Reviewer: ```suggestion
| 30533 | TCP/UDP (QUIC)      | Always public                                           | Subspace networking P2P    | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li></ul>                                                                        | Connecting to the rest of the network and retrieving archival history, not required for Pulsar and Space Acres because the node and the farmer share the Subspace networking stack there. |
```

| 40333 | TCP (QUIC)          | Always public                                           | Subspace networking P2P    | <ul><li>[Advanced CLI Node (Domain)](../../../category/operators-and-nominators)</li></ul>                                                                           | Connecting to the rest of the domain network for transaction gossipping.      |

> | 9944  | TCP (HTTP/WebSocket)| Public: <ul><li>RPC server</li></ul> Private: <ul><li>Internal node-to-farmer connections</li></ul> | Consensus RPC port         | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li></ul>                                                                        | RPC server: Expose various details about blockchain (real-time and historical events, storage, transactions, etc.) as well as submitting transactions to the network (balance transfers, staking, etc.). <br /> Farmer connections: Farmer connects in unsafe mode and receives notifications about slot challenges for block/vote production, archival history extension events, querying of various chain details. Nodes that process farmer connections should never be exposed publicly as public exposure of unsafe APIs can be used to not only prevent farmer from working, but even block node sync completely. |

Reviewer: ```suggestion
| 9944  | TCP (HTTP/WebSocket)| Public: <ul><li>RPC server</li></ul> Private: <ul><li>Internal node-to-farmer connections</li></ul> | Consensus RPC port         | <ul><li>[Advanced CLI Node (Consensus)](../../../category/advanced-cli-recommended)</li></ul>                                                                        | RPC server: Exposes various details about the blockchain (real-time and historical events, storage, transactions, etc) as well as submitting transactions to the network (balance transfers, staking, etc). <br /><br /> Farmer connections: The farmer connects in unsafe mode and receives notifications about slot challenges for block/vote production, archival history extension events, querying of various chain details. Nodes that process farmer connections should never be exposed publicly as public exposure of unsafe APIs can be used to not only prevent farmer from working, but even block node sync completely. |
```

> | 9945  | TCP (HTTP/WebSocket)| Public: <ul><li>RPC server</li></ul>                     | Domain RPC port            | <ul><li>[Advanced CLI Node (Domain)](../../../category/operators-and-nominators)</li></ul>                                                                           | RPC server: Expose various details about blockchain (real-time and historical events, storage, transactions, etc.) as well as submitting transactions to the network (balance transfers, interacting with smart contracts, etc.). |

Reviewer: ```suggestion
| 9945  | TCP (HTTP/WebSocket)| Public: <ul><li>RPC server</li></ul>                     | Domain RPC port            | <ul><li>[Advanced CLI Node (Domain)](../../../category/operators-and-nominators)</li></ul>                                                                           | RPC server: Expose various details about blockchain (real-time and historical events, storage, transactions, etc.) as well as submitting transactions to the network (balance transfers, interacting with smart contracts, etc). |
```
